pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -843,5 +843,37 @@ public void setSchemaAutoUpdateCompatibilityStrategy(@PathParam("tenant") String
         internalSetSchemaAutoUpdateCompatibilityStrategy(strategy);
     }
 
+    @GET
+    @Path("/{tenant}/{namespace}/schemaValidationEnforced")
+    @ApiOperation(value = "Get schema validation enforced flag for namespace. If the flag is set to true, when "
+                            + " a producer without a schema attempts to produce to a topic with schema in this"
+                            + " the namespace, the producer will be failed to connect. PLEASE be carefully on"
```

Comment:
remove "the" before "namespace", since we have "this" before.
also how about split "value" into "value" and "notes" as line 832?